### ISSUE TITLE
Resolution: PowerBI "key didn't match" error

### ISSUE THREAD

**Issue by AGSHoward** [BrentOzarULTD/SQL-Server-First-Responder-Kit#1691]
**Do you want to request a *feature* or report a *bug*?**
If you're reporting a bug, include the version number of the script you're using. If it's not the current version, upgrade to the current version and test that before reporting a bug - we fix a lot of stuff in each new build.

Bug with PowerBI template as posted on github from May 28th

**What is the current behavior?**
When you first load the PBI file and connect to your data source you get an error "The key didn't match any rows in the table", your not able to resolve this issue here and can only close the window on which the PBI file will load but without data.

![pbi refresh](https://user-images.githubusercontent.com/41490413/43024216-70b5e7c2-8c22-11e8-9534-0b5d4ec84546.PNG)

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem via http://sqlfiddle.com**

- Open the PBI file and enter your data source as asked
- The error above will display

To resolve take these actions:

- Click on Edit Queries in Power BI top menu
- Click on each query on the left bar - two will show a triangle with an !

![editqueries](https://user-images.githubusercontent.com/41490413/43024535-8e6ab896-8c23-11e8-94d1-7c32bae2c4f7.PNG)

- On each query with an error modify the Navigation settings by clicking on the gear icon

![settingsgear](https://user-images.githubusercontent.com/41490413/43024680-d682c754-8c23-11e8-888b-03a7859eda75.PNG)

- and modify the table to connect to

![navigation](https://user-images.githubusercontent.com/41490413/43024873-7deff688-8c24-11e8-97c1-815228727493.PNG)

- Do this for each error reference and ensure the names match up {Deltas to Deltas, Actuals to Actuals}
- Click on Apply and Close and the queries will now resolve and your data will be rendered.

**What is the expected behavior?**
The report should now load as the example powerbi report does.

**Comment by BrentOzar** [comment]
OK, just to be clear - you're not proposing a change to the Power BI doc, just explaining how you fixed this on your system? (I'm guessing you're using a different database name, perhaps? It's not clear what the issue was on your system.)

**Comment by AGSHoward** [comment]
Correct, I used a database name of FirstResponderKit and a FQDN for the server instance. The job references that database and all of the First Responder sprocs and views are in that database. So nothing outside the ordinary from the details of deployment per your doc, it all works bar those two queries in PowerBI.

Totally agree could be peculiar to my system, hence not proposing you change anything but maybe have a section somewhere of known/potential issues. 

Erik helped me out initially, I redid everything, even dropped and started again from scratch. I got it going by making this change - so just looking to pay it forward to the community.
